Software/Python/setup.py

```diff
@@ -12,7 +12,26 @@
     name="gopigo3",
     description="Drivers and examples for using the GoPiGo3 in Python",
     author="Dexter Industries",
-    url="http://www.dexterindustries.com/GoPiGo/",
+    url="http://www.dexterindustries.com/GoPiGo3/",
     py_modules=['gopigo3','easygopigo3','I2C_mutex'],
     #install_requires=open('requirements.txt').readlines(),
+    version='1.0.0',
+    long_description=" \
+    This python package includes two main modules for development: \n \
+    1. gopigo3 - This package is a low level module for controlling the GoPiGo3. \n \
+    2. easygopigo3 - This package is a higher-level module built on top of the gopigo3 module which is more user friendly.  \n \ ",
+    license='MIT',
+    classifiers=[
+        'Development Status :: 5 - Production/Stable',
```

Comment:
Referring to `classifiers` list : what do you think of the `classifiers` found [here](https://github.com/RobertLucian/GoPiGo3/blob/feature/pip-package/Software/Python/setup.py)?